source/ui/IDEAS/IDEASPersistentView.cpp

```diff
@@ -88,132 +88,125 @@ IDEASPersistentView::IDEASPersistentView(QWidget *parent) :
-				jjSlitsHorizOpen_->setMinimumHeight(15);
-				jjSlitLayout_->addWidget(jjSlitsHorizOpen_,0,0,1,1);
-				jjSlitsHorizClose_ = new AMControlMoveButton();
-				jjSlitsHorizClose_->setControl(IDEASBeamline::ideas()->jjSlitHGap());
-				jjSlitsHorizClose_->setStepSizes(QList<double>() << 0.1 << 0.5 << 1 << 5);
-				jjSlitsHorizClose_->setStepSizeIndex(2);
-				jjSlitsHorizClose_->setDirectionReversed(true);
-				jjSlitsHorizClose_->setIcon(iconHorizontalClose);
-				jjSlitsHorizClose_->setIconSize(QSize(15, 15));
-				jjSlitsHorizClose_->setToolButtonStyle(Qt::ToolButtonIconOnly);
-				jjSlitsHorizClose_->setMinimumWidth(15);
-				jjSlitsHorizClose_->setMinimumHeight(15);
-				jjSlitLayout_->addWidget(jjSlitsHorizClose_,1,0,1,1);
-
-
-				jjSlitsUp_ = new AMControlMoveButton();
-				jjSlitsUp_->setControl(IDEASBeamline::ideas()->jjSlitVCenter());
-				jjSlitsUp_->setStepSizes(QList<double>() << 0.1 << 0.5 << 1 << 5);
-				jjSlitsUp_->setStepSizeIndex(2);
-				jjSlitsUp_->setIcon(iconUp);
-				jjSlitsUp_->setIconSize(QSize(15, 15));
-				jjSlitsUp_->setToolButtonStyle(Qt::ToolButtonIconOnly);
-				jjSlitsUp_->setMinimumWidth(15);
-				jjSlitsUp_->setMinimumHeight(15);
-				jjSlitLayout_->addWidget(jjSlitsUp_,2,2,1,1);
-				jjSlitsDown_ = new AMControlMoveButton();
-				jjSlitsDown_->setControl(IDEASBeamline::ideas()->jjSlitVCenter());
-				jjSlitsDown_->setStepSizes(QList<double>() << 0.1 << 0.5 << 1 << 5);
-				jjSlitsDown_->setStepSizeIndex(2);
-				jjSlitsDown_->setDirectionReversed(true);
-				jjSlitsDown_->setIcon(iconDown);
-				jjSlitsDown_->setIconSize(QSize(15, 15));
-				jjSlitsDown_->setToolButtonStyle(Qt::ToolButtonIconOnly);
-				jjSlitsDown_->setMinimumWidth(15);
-				jjSlitsDown_->setMinimumHeight(15);
-				jjSlitLayout_->addWidget(jjSlitsDown_,3,2,1,1);
-
-
-				jjSlitsLeft_ = new AMControlMoveButton();
-				jjSlitsLeft_->setControl(IDEASBeamline::ideas()->jjSlitHCenter());
-				jjSlitsLeft_->setStepSizes(QList<double>() << 0.1 << 0.5 << 1 << 5);
-				jjSlitsLeft_->setStepSizeIndex(2);
-				jjSlitsLeft_->setIcon(iconLeft);
-				jjSlitsLeft_->setIconSize(QSize(15, 15));
-				jjSlitsLeft_->setToolButtonStyle(Qt::ToolButtonIconOnly);
-				jjSlitsLeft_->setMinimumWidth(15);
-				jjSlitsLeft_->setMinimumHeight(15);
-				jjSlitLayout_->addWidget(jjSlitsLeft_,0,2,1,1);
-				jjSlitsRight_ = new AMControlMoveButton();
-				jjSlitsRight_->setControl(IDEASBeamline::ideas()->jjSlitHCenter());
-				jjSlitsRight_->setStepSizes(QList<double>() << 0.1 << 0.5 << 1 << 5);
-				jjSlitsRight_->setStepSizeIndex(2);
-				jjSlitsRight_->setDirectionReversed(true);
-				jjSlitsRight_->setIcon(iconRight);
-				jjSlitsRight_->setIconSize(QSize(15, 15));
-				jjSlitsRight_->setToolButtonStyle(Qt::ToolButtonIconOnly);
-				jjSlitsRight_->setMinimumWidth(15);
-				jjSlitsRight_->setMinimumHeight(15);
-				jjSlitLayout_->addWidget(jjSlitsRight_,1,2,1,1);
-
-
-
-
-
-			jjSlitGroupBox_->setLayout(jjSlitLayout_);
-
-
+	QGroupBox *jjSlitGroupBox = new QGroupBox;
```

Comment:
Hilariously, no.  The JJSlits for IDEAS actually work differently under the hood.  You directly tell it to move the gap and center whereas other JJ slit implementations let you control the actual blades.